### PR TITLE
COG-215 Add User Button to UserList page

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**
- [x] Refactor Card Component  (button names as props)

- [x] Change enableSave prop name to disableActionBtn

- [x] Add "+ ADD A USER"Button to UserList card

### CODE REVIEW

File: app/javascript/common/Card.js [hunk 1]
```diff
@@ -3,16 +3,22 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import { Button } from 'react-wood-duck';
 
-function renderCancelSaveButtons(props, onCancel, enableSave, onSave) {
+function renderCancelSaveButtons(
```
Comment (RamuKammagani-osi) [inline app/javascript/common/Card.js]: can we rename this function name something like renderCardButtons(),  
renderCancelSaveButtons () is a bit confusing

**Comment by kelfink** [comment]
Code Climate says we have to break up those longer methods

**Comment by Sushmareddy-OSI** [comment]
I think, we already discussed about this. 
Those issues are from Card component which is a react-wood-duck component, down the road we will push that component to react-wood-duck. I don't think we need to refactor it now!!

**Comment by kelfink** [comment]
We will ignore the code climate  warnings on Card, as we don't want to stray far from the react-wood-duck version which will ultimately be the new version

**Comment by kelfink** [comment]
retest this please

**Pull request merged**